src/jsMain/kotlin/app/softwork/bootstrapcompose/Styling.kt

```diff
@@ -0,0 +1,386 @@
+
+    public operator fun invoke(f: Background.() -> Unit) {
+        this.f()
+    }
+
+    internal fun generateClassStrings(): Array<String> {
+        val classes: MutableList<String> = mutableListOf()
+
+        color?.let { color ->
+            classes += color.background()
+            if (gradient) {
+                classes += "bg-gradient"
+            }
+        }
+
+        return classes.toTypedArray()
+    }
+}
+
+public class Text {
+    public var color: Color? = null
+    private val alignments: MutableList<AlignSpec> = mutableListOf()
+    public var wrap: Wraps? = null
+    public var wordBreak: Boolean = false
+    public var transform: Transform? = null
+    public var size: Int? = null
+    public var weight: Weight? = null
+    public var style: Style? = null
+    public var lineHeight: LineHeight? = null
+    public var monospace: Boolean = false
+    public var reset: Boolean = false
+    public var muted: Boolean = false
+    public var decoration: Decoration? = null
+
+    public operator fun invoke(f: Text.() -> Unit) {
+        this.f()
+    }
+
+    public enum class Alignment(private val value: String) {
+        Start("start"),
+        Center("center"),
+        End("end");
+
+        override fun toString(): String {
+            return value
+        }
+    }
+
+    public enum class Wraps(private val value: String) {
+        Wrap("text-wrap"),
+        NoWrap("text-nowrap");
+
+        override fun toString(): String {
+            return value
+        }
+    }
+
+    public enum class Transform(private val value: String) {
+        Lowercase("text-lowercase"),
+        Uppercase("text-uppercase"),
+        Capitalized("text-capitalize");
+
+        override fun toString(): String {
+            return value
+        }
+    }
+
+    public enum class Weight(private val value: String) {
+        Bold("fw-bold"),
+        Bolder("fw-bolder"),
+        Normal("fw-normal"),
+        Light("fw-light"),
+        Lighter("fw-lighter");
+
+        override fun toString(): String {
+            return value
+        }
+    }
+
+    public enum class Style(private val value: String) {
+        Italic("fst-italic"),
+        Normal("fst-normal");
+
+        override fun toString(): String {
+            return value
+        }
+    }
+
+    public enum class LineHeight(private val value: String) {
+        Smallest("lh-1"),
+        Small("lh-sm"),
+        Base("lh-base"),
+        Large("lh-large");
+
+        override fun toString(): String {
+            return value
+        }
+    }
+
+    public enum class Decoration(private val value: String) {
+        Underline("text-decoration-underline"),
+        LineThrough("text-decoration-line-through"),
+        None("text-decoration-none");
+
+        override fun toString(): String {
+            return value
+        }
+    }
+
+    public class AlignSpec(internal val alignment: Alignment, public var breakpoint: Breakpoint? = null)
```

Comment:
why did you hide/publish one attribute, but not the other?

Reply:
not intentional. I fixed it in the latest commit.